packages/flutter_tools/test/general.shard/generate_localizations_test.dart

```diff
@@ -2142,7 +2175,11 @@ import 'output-localization-file_en.dart' deferred as output-localization-file_e
           throwsA(isA<L10nException>().having(
             (L10nException e) => e.message,
             'message',
-            contains('Resource attribute "@genderSelect" was not found'),
+//             contains('''
```

Comment:
delete comment?

Reply:
Whoops, forgot to put a todo there. I'll need the change in an upcoming PR.

Reply:
We generally do not allow commented code, and prefer using git to track work we intend to land later.